config/dev.exs

```diff
@@ -1,7 +1,7 @@
 use Mix.Config
 
 config :kafka_ex,
-  brokers: [{"ip-10-0-0-49.ec2.internal", 9092}, {"ip-10-0-0-248.ec2.internal", 9092}],
+  brokers: [{"ip-10-0-0-166.ec2.internal", 9092}],
```

Comment:
I'm going to switch to dns for kafka in a separate PR. this PR other than that should be good to go.